components/desktop/pulseaudio/pulseaudio.p5m

```diff
@@ -24,15 +24,8 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 <transform file path=.*/etc/.* -> default preserve true>
 
-link path=etc/bash_completion.d/pacat target=pulseaudio
-link path=etc/bash_completion.d/pacmd target=pulseaudio
-link path=etc/bash_completion.d/pactl target=pulseaudio
-link path=etc/bash_completion.d/padsp target=pulseaudio
-link path=etc/bash_completion.d/paplay target=pulseaudio
-link path=etc/bash_completion.d/parec target=pulseaudio
-link path=etc/bash_completion.d/parecord target=pulseaudio
-link path=etc/bash_completion.d/pasuspender target=pulseaudio
-file path=etc/bash_completion.d/pulseaudio
+file files/pulse_exec_attr path=etc/security/pulseaudio
```

Comment:
Another issue is that pulseaudio should be run with pfexec after this. Also it seems proc_prioup is missing.